### PR TITLE
Fix Issue 17536 - dlangspec.pdf: Unittests documentation uses HTML

### PR DESCRIPTION
**Pull request by andralex**

### CODE REVIEW

File: spec/unittest.dd [hunk 1]
```diff
@@ -121,14 +121,13 @@ unittest
 
 $(P The above will generate the following documentation:)
 
+$(RAWHTML
```
Comment (wilzbach) [inline spec/unittest.dd]: Needs to be put in `ebook.ddoc` too:

```
/dev/shm/dtest/work/repo/dlang.org/web/dlangspec.html:20910:<!--UNDEFINED MACRO: "RAWHTML"-->
/dev/shm/dtest/work/repo/dlang.org/web/dlangspec.html:20948:<!--UNDEFINED MACRO: "RAWHTML"-->
```

File: spec/unittest.dd [hunk 2]
```diff
@@ -176,25 +176,24 @@ unittest
...
 <b>Examples:</b><br>
 code sample generated
-<pre class="d$(UNDERSCORE)code">
+<pre class="d_code">
 <font color="blue">assert</font>(<u>add</u>(1, 1) == 2);
 </pre>
 <br><br><b>Examples:</b><br>
 code sample generated, even if it is empty or only includes comments
-
 <pre class="d$(UNDERSCORE)code">
```
Comment (wilzbach) [inline spec/unittest.dd]: You missed that one.

File: spec/unittest.dd (hunk 1 above)
Comment (wilzbach) [reply spec/unittest.dd]: I appended this.

**Comment by andralex** [comment]
@wilzbach thx!

**Pull request merged**